scripts/blacklist.lua

```diff
@@ -0,0 +1,24 @@
+local blacklist = {}
+
+blacklist.add_remote_interface = function()
+  remote.add_interface("PipeVisualizer", {
+    --- @param entity_names string[]
+    blacklist = function(entity_names)
+      if type(entity_names) ~= "table" then return end
+      for _, entity_name in pairs(entity_names) do
+        global.blacklist[entity_name] = true
+      end
+    end,
+  })
+end
+
+local function reset(e)
```

Comment:
Missing type annotation.

Reply:
I did the following, not sure if its correct.
```lua
---@param e ConfigurationChangedData?
local function reset(e)
```